frontend/packages/integration-tests-cypress/views/snapshot.ts

```diff
@@ -0,0 +1,11 @@
+export namespace SnapshotDetails {
+  export const name = 'dd[data-test-selector="details-item-value__Name"]';
+  export const namespace = 'dd[data-test-selector="details-item-value__Namespace"] a';
+  export const pvc = 'dd[data-test-selector="details-item-value__PVC"] a';
+  export const vsc = 'dd[data-test-selector="details-item-value__VSC"] a';
+  export const sc = 'dd[data-test-selector="details-item-value__SC"] a';
+}
```

Comment:
Please see previous comment about changing `data-test-selector=` to `data-test=`.